code/game/objects/items/food/cake.dm

```diff
@@ -609,3 +609,22 @@
 	icon_state = "plumcakeslice"
 	tastes = list("cake" = 5, "sweetness" = 1, "plum" = 2)
 	foodtypes = GRAIN | DAIRY | FRUIT | SUGAR
+
+/obj/item/food/cake/wedding
+	name = "wedding cake"
+	desc = "An expensive, multi-tiered cake."
+	icon_state = "weddingcake"
+	food_reagents = list(
+		/datum/reagent/consumable/nutriment = 40,
+		/datum/reagent/consumable/nutriment/vitamin = 10
```

Comment:
Not for any balance reasons or anything, but you might want to add some sugar to the food_reagents, since the recipe calls for it.

Reply:
30u of sugar added to it